xml/geo_manage_i.xml

```diff
@@ -28,161 +28,152 @@
-   <para>
-    Assuming that you want to manually move <literal>&ticket1;</literal> from
-    site <literal>&cluster1;</literal> (with the virtual IP
-    <literal>192.168.201.100</literal>) to site <literal>&cluster2;</literal>
-    (with the virtual IP <literal>192.168.202.100</literal>), proceed as
-    follows:
-   </para>
+  <procedure xml:id="pro.ha.geo.manage.manual.tickets">
+   <title>Managing Manual Tickets</title>
+   <para> Let us assume that the (manually managed) ticket
+     <literal>&ticket3;</literal> had been granted to site
+     <literal>&cluster1;</literal> (with the virtual IP
+     <literal>192.168.201.100</literal>. This site cannot be reached at the
+    moment. Site <literal>&cluster2;</literal> (with the virtual IP
+     <literal>192.168.202.100</literal>) is still available. </para>
+   <para> If you manage to contact a local administrator on
+     <literal>&cluster1;</literal>, you can verify if the site is really down.
+    In that case, the ticket on <literal>&cluster1;</literal> cannot be revoked
+    and the resources depending on this ticket cannot be started on
+     <literal>&cluster2;</literal> unless you manually grant the ticket
+     <literal>ticket-nfs</literal> to <literal>&cluster2;</literal>. To do so,
+    proceed as follows: </para>
+   <step>
+    <para> Log in to <literal>&cluster2;</literal>.</para>
+   </step>
    <step>
-    <para>
-     Set <literal>&ticket1;</literal> to standby with the following command:
-    </para>
-<screen>&prompt.root;<command>crm_ticket</command> -t &ticket1; -s</screen>
+    <para> Grant <literal>ticket-nfs</literal>to site
+      <literal>&cluster2;</literal> using the <option>-F</option> option: </para>
+    <screen>&prompt.root;<command>booth</command> grant -F ticket-nfs</screen>
+    <para>You will see a warning that the same ticket might be granted to
+     another site, but the command will be executed.</para>
    </step>
    <step>
-    <para>
-     Wait for any resources that depend on <literal>&ticket1;</literal> to be
-     stopped or demoted cleanly.
-    </para>
+    <para>Check the result with:</para>
+    <screen>&prompt.root;<command>booth</command> list</screen>
+    <para>It should show <literal>&cluster2;</literal> as ticket owner for
+      <literal>ticket-nfs</literal> now. All resources that depend on this
+     ticket will be started on <literal>&cluster2;</literal>.</para>
    </step>
    <step>
-    <para>
-     Revoke <literal>&ticket1;</literal> from site
-     <literal>&cluster1;</literal> with:
-    </para>
-<screen>&prompt.root;<command>booth</command> revoke -s 192.168.201.100 &ticket1;</screen>
+    <para> Before trying to get <literal>&cluster1;</literal> up and running
+     again, set <literal>ticket-nfs</literal> to standby on <literal>&cluster2;</literal> (to
+     prevent that resources will be started on both sites in parallel): </para>
+    <screen>&prompt.root;<command>crm_ticket</command> -t ticket-nfs --standby</screen>
```

Comment:
Depending on the needs, if we want to keep the resources running on cluster2, we should not do this since it'd unnecessarily stop the resources and introduce service down time again. I think something like this probably makes more sense:

"Before tying to bring cluster1 back into the geo cluster, again make sure the ticket is revoked and resources are not hosted there."

For the "standby" and "activate" commands, we can introduce them as parts of the normal procedure when doing manual switch-over when all the geo cluster sites are healthy.